addons/portal/static/src/scss/portal.scss

```diff
@@ -161,7 +161,7 @@ hr {
 .input-group {
     flex-flow: row nowrap;
 }
-.list-group-item:not([class*="list-group-item-"]) {
+.list-group-item:not([class*="list-group-item-"]):not(.active) {
```

Comment:
Well spotted. But different commit for this :)

Reply:
:+1:  Was not that hard to spot tbh, I had a big black text on a turquoise background :yum: 
Alight ill make another commit

Reply:
Done